.changeset/quick-items-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": minor
+---
+
+add plugins
```

Comment:
The changeset should ideally be more specific, i.e mention what plugins.

Reply:
noted for next time @haammar-ledger